frontend/src/Pages/RegisterByDate.css

```diff
@@ -35,56 +29,63 @@
   display: flex;
   align-self: flex-start;
 }
+
+.calendar {
+  display: flex;
+  align-items: center;
+}
+.lowerArea {
+  display: flex;
+  flex-direction: column;
+  justify-content: baseline;
+  align-self: center;
+}
 .howManyArea {
   display: flex;
+  align-self: center;
   flex-direction: column;
-  margin: auto;
-  margin: 5%;
-  padding: 30%;
-  padding-bottom: 10%;
-
-  /* Gray 6 */
+  margin-bottom:15%;
+  padding-bottom:70%;
+  padding-right:60%;
+  padding-left:60%;
   background: #f2f2f2;
   border-radius: 4px;
 }
-.calander {
-  display: flex;
-  margin: 40%;
-}
-.lowerArea {
+.amount{
+  color: #00406b;
   display: flex;
-  flex-direction: column;
-  align-self: end;
-  justify-content: flex-end;
+  font-size: 4vw;
+  align-self: center;
+  font-family: "Arial";
+  font-weight: bold; 
+  padding-left: 100%;
+  padding-right: 100%;
+  white-space: nowrap;
 }
-
 .attendance {
-  /* See who registered */
   font-family: "Arial";
+  margin-bottom:0%;
   font-style: normal;
   font-weight: 400;
-  font-size: 18px;
-  line-height: 120%;
+  font-size: 5vw;
   display: flex;
-  align-self: flex-end;
   color: #2b8fff;
+  text-justify: distribute;
+  padding-left: 0%;
+  padding-right: 0%;
+  align-self: center;
+  justify-self: baseline;
+  white-space: nowrap;
 }
 .continue {
   display: flex;
-  align-self: flex-end;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
-  margin: 0px 10px;
   font-family: "Arial";
   font-style: normal;
-  font-weight: 700;
-  font-size: 18px;
-  display: flex;
+  font-weight: bold;
+  font-size: 5vw;
```

Comment:
The `vw` unit is a relative unit depending on the viewport's width - the width of the window.

of mobile, it will be too small.

consider using the absolute unit like `px`